app/models/katello/pool.rb

```diff
@@ -9,6 +9,8 @@ class Pool < Katello::Model
     has_many :activation_keys, :through => :pool_activation_keys, :class_name => "Katello::ActivationKey"
     has_many :pool_activation_keys, :class_name => "Katello::PoolActivationKey", :dependent => :destroy, :inverse_of => :pool
 
+    scope :organization_id, -> (org_id) { joins(:subscription).where("#{Katello::Subscription.table_name}.organization_id = ?", org_id) }
```

Comment:
Nitpick on the naming, something like `in_org` might be more descriptive (e.g. https://github.com/Katello/katello/blob/master/app/models/katello/product.rb#L48)